src/light_client.rs

```diff
@@ -350,12 +349,11 @@ pub async fn process_block(
 
 	metrics.record(MetricValue::DHTPutSuccess(dht_insert_success_rate as f64))?;
 
-	let dht_put_time_elapsed = begin.elapsed()?;
+	let dht_put_time_elapsed = begin.elapsed();
 	info!(
 		block_number,
-		"partition_dht_insert_time_elapsed" = dht_put_time_elapsed.as_secs_f64(),
```

Comment:
If we have a dashboard in data dog or whatever tool we use it that depends on those keys, this will break it. I'm not sure how to confidently perform changes like this one. One way would could be to have a list of constant keys allowed to be used in structured logs and to understand that any change is breaking.

Reply:
I can return it back if it is the case, but I think that logs are acting like telemetry here. WDUT?

Reply:
I think we have structured logs for reason, one of them can be indexing and interpreting it somehow, like listing top entries with longest dht insert time. But I'm not sure in this exact case so it might be harmless to make a change, hope someone else knows more about it.

Reply:
So far I can't find anything related in our org which uses it. I think we're free to assume it will be okay to change format or even remove this log in future